next-tavla/src/Board/scenarios/Table/components/TableHeader/index.tsx

```diff
@@ -10,8 +9,8 @@ function TableHeader({
     walkingDistance?: TWalkingDistance
 }) {
     return (
-        <div className="flex flex-row justify-between items-center mb-2">
-            <h1 className={classes.heading}>{heading}</h1>
+        <div className="flex flex-row justify-between items-center mb-2 min-h-[2em]">
+            <h1 className="text-[1.5em] font-semibold m-0">{heading}</h1>
```

Comment:
jeg regner med det er bevist at dette er arbitrary data

Reply:
ja, men vi kan lage en `text-em-xl` for `1.5em` evt - vi bare bruker den ikke noen andre steder. `h-[2em]` brukes flere plasser, så kan være legger den i configen